api-examples/pretrain-paired.py

```diff
@@ -72,11 +74,14 @@ def train():
                         help="Path or url of the dataset cache")
     parser.add_argument("--d_model", type=int, default=512, help="Model dimension (and embedding dsz)")
     parser.add_argument("--d_ff", type=int, default=2048, help="FFN dimension")
-    parser.add_argument("--d_k", type=int, default=None, help="Dimension per head.  Use if num_heads=1 to reduce dims")
+    parser.add_argument("--d_k", type=int, default=64, help="Dimension per head.  Use if num_heads=1 to reduce dims")
```

Comment:
add a new option to reduction layer, and document it.

Reply:
I edont think we want d_k on by default